src/pin-code/pin-code.js

```diff
@@ -33,7 +33,12 @@ export default class PinCode extends React.Component<PropsT, StateT> {
       if (inputRef && inputRef.focus) inputRef.focus();
     }
   }
-
+  getMaskStyle(i: number) {
+    if (this.props.values[i] == '') return this.props.values[i];
+    if (typeof this.props.mask === 'boolean')
+      return this.props.mask ? '*' : this.props.values[i];
```

Comment:
Can we render a solid circle if boolean `true` value is provided, rather than the star symbol?

Reply:
This may be done best by exploring the `type="password"` attribute